client/components/user-home.js

```diff
@@ -1,16 +1,52 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {connect} from 'react-redux'
+import moment from 'moment'
+import {Link} from 'react-router-dom'
+import {getAllEntries} from '../store'
 
 /**
  * COMPONENT
  */
 export const UserHome = props => {
-  const {email} = props
+  const {user} = props
 
   return (
     <div>
-      <h3>Welcome, {email}</h3>
+      <h3>Welcome, {user.name}</h3>
+      <div className="row">
+        <h4>Account information</h4>
+        <p>
+          <b>Name: </b>
+          {user.name}
+        </p>
+        <p>
+          <b>Email: </b>
+          {user.email}
+        </p>
+      </div>
+      <div className="row">
+        <h4>History</h4>
+        <p>
+          <b>Member since: </b>
+          {moment(user.createdAt).format('MMMM Do YYYY')}
```

Comment:
Is this supposed to say DD instead of Do?

Reply:
No,  Do adds the "th" after the date as in:
moment().format("MMM Do YY")  // Mar 25th 19